pystiche_papers/johnson_alahi_li_2016/_nst.py

```diff
@@ -70,6 +70,8 @@ def training(
     style_image = batch_up_image(style_image, loader=content_image_loader)
 
     if impl_params:
+        # https://github.com/pmeier/fast-neural-style/blob/813c83441953ead2adb3f65f4cc2d5599d735fa7/slow_neural_style.lua#L111
+        # A preprocessor is used in the implementation, which is not documented in the paper.
```

Comment:
We might wanna discuss this. Although they did not mention it specifically in their paper, the usage of the VGG weights implies this pre- and postprocessing. I'll think this through and open an issue for this.